Resources/Maps/saltern.yml

```
@@ -6161,7 +6161,7 @@ entities:
         beaker:
           temperature: 293.15
           canReact: True
-          maxVol: 50
+          maxVol: 60 #Carpmosia-edit - reagentStorage&Transfer
```

```suggestion
          maxVol: 50
```
I don't think we should touch upstream maps, not until we implement toolshed patching